scripts/SDM/BRT/fitBRT.jl

```diff
@@ -0,0 +1,66 @@
+using EvoTrees
+using CSV
+using DataFrames
+using JSON
+using SpeciesDistributionToolkit
+
+include("io.jl")
+
+function main()
+    _crs = "EPSG:6622"
+
+    RUNTIME_DIR = ARGS[1]
+    inputs = read_inputs_dict(RUNTIME_DIR)
+    bbox = (left=-2316297.0, right=-1971146.0, bottom=1015207.0, top=1511916.0)
```

Comment:
FYI SimpleSDMLayers release from this morning uses WGS84 for bbox now - so this needs to be changed

Reply:
well at least it isnt hardcoded anymore